### ISSUE TITLE
[Warning] Add-Type command still touches disk

### ISSUE THREAD

**Issue by GetRektBoy724** [SecureThisShit/Invoke-Sharpcradle#1]
Just a warning, `Add-Type` command still touches the disk cause it still needs to compile it (Just-In-Time).
The better way is just compile it ahead of time, and then convert it into base64, and use Reflection.Assembly to load the base64.

**Comment by S3cur3Th1sSh1t** [comment]
Yeah I knew that already. But the part on disk was never caught for me in any environment. `Assembly::load` would not work here as AMSI would catch the code when it's loaded. I won't provide signature updates here and therefore it will stay like it is.

But the warning can stay alive for others using it.

**Comment by GetRektBoy724** [comment]
ahhh ok then :smile:
